dmx_client/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.8'
+version: '3.3'
```

Comment:
Wersja ta działa na raspberry PI, ale trzeba zaktualizować jeszce requirements i w Dockerfile zjechac do wersji pythona która bedzie dzialac na raspberry. Wtedy dopiero sprawdzimy czy możemy z Dockera korzystać

Reply:
Koniec końców nie korzystamy z Docker na rpi, to zrobimy w kolejnym fix'sie